google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -112,6 +112,7 @@ public BucketInfo apply(com.google.api.services.storage.model.Bucket pb) {
 
     private Boolean isUniformBucketLevelAccessEnabled;
     private Long uniformBucketLevelAccessLockedTime;
+    private String publicAccessPrevention;
```

Comment:
@shaffeeullah @tritone are other languages introducing Enums for this value or just keeping it as a string?

Reply:
In python and node we used strings with provided constants for "enforced" and "unspecified". So at the very least, we should provide these constants in the library surface (this is not done currently looks like).

If enums are more typical in this library (vs. node or python) we should feel free to use an enum here as well.

Reply:
Thanks @tritone. It would be helpful @athakor to introduce enums to reduce need for string literals.

Reply:
done